models/Page.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+const slug = require('mongoose-slug-generator');
+
+const pageSchema = new mongoose.Schema({
+  siteId: { type: mongoose.Schema.Types.ObjectId, ref: 'Site' },
```

Comment:
Can we rename `siteId` to `site`? So when we need to populate it, we can use `.populate('site')`, the same with `authorId` we can rename it to `author`